pages/mi/profiles/sxknnn.md

```diff
@@ -0,0 +1,20 @@
+Sujana Kabir
+=======
+| Location            | Timezone      | OS                 |
+| ------------------- | ------------- | ------------------ |
+| Arlington, Texas    | GMT -6        | Windows 11 |
+
+About Me
+----------
+I am an **Android Developer** with a passion of creating innovative and user friendly mobile applications.
+
+I use these languages for app development:
+<ul>
+  <li> JavaScript </li>
+  <li> Java </li>
+</ul>
```

Comment:
Please avoid HTML here as we are practicing with Markdown. I will look into fixing the profile examples.

Reply:
agree